internal/volumeserver/volumesnapshot_delete.go

```diff
@@ -0,0 +1,28 @@
+// SPDX-FileCopyrightText: 2022 SAP SE or an SAP affiliate company and IronCore contributors
+// SPDX-License-Identifier: Apache-2.0
+
+package volumeserver
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/ironcore-dev/ceph-provider/internal/utils"
+	iri "github.com/ironcore-dev/ironcore/iri/apis/volume/v1alpha1"
+	"github.com/pkg/errors"
```

Comment:
```suggestion
	"errors"
```
Use standard library "`errors`" package instead.